eng/config/PublishData.json

```diff
@@ -182,6 +182,17 @@
       "vsBranch": "main",
       "vsMajorVersion": 17
     },
+    "release/dev17.0-preview1-nopia-vs-deps": {
+      "nugetKind": [
+        "Shipping",
+        "NonShipping"
+      ],
+      "version": "4.0.*",
+      "packageFeeds": "default",
+      "channels": [],
+      "vsBranch": "feature/d17initial",
+      "vsMajorVersion": 17
+    },
```

Comment:
Does this need to be updated in main? Also, is this going to cause conflicts with multiple branches publishing at once with potentially same version?

Reply:
@genlu yeah this change needs to be made in main, PublishData.json is only ever read from main

For publishing, should we also remove publishing for the original preview1 branch?  Or do we need to publish that still

Reply:
> PublishData.json is only ever read from main

Ah, right, I forgot about that.

> Also, is this going to cause conflicts with multiple branches publishing at once with potentially same version?

Yes. I'm hoping this is gonna be a one-off things for this branch so it should be manageable. FWIW, other teams (e.g. editor) that are doing dual insertions (to both d17intial and MergedInterops) are having the same problem, you have to check the dependecies of their package to know for sure.

> For publishing, should we also remove publishing for the original preview1 branch

there's still need to publish from that branch, and I think it will be our preview1 branch going forward even after merged interops are in d17intial, right?